committing.rst

```diff
@@ -5,212 +5,155 @@ Accepting Pull Requests
-there must be a good reason for the breakage of code the change will cause
-(and it **will** break someone's code). If you are unsure if the breakage
-is worth it, ask on python-dev.
-
-Third, ensure the patch is attributed correctly with the contributor's
-name in ``Misc/ACKS`` if they aren't already there (and didn't add themselves
-in their patch) and by mentioning "Patch by <x>" in the ``Misc/NEWS.d`` entry
-and the check-in message. If the patch has been heavily modified then "Initial
-patch by <x>" is an appropriate alternate wording.  GitHub now supports `multiple
-authors <https://help.github.com/articles/creating-a-commit-with-multiple-authors/>`_
-in a commit. Add ``Co-authored-by: name <name@example.com>`` at the end of the commit message.
-
-If you omit correct attribution in the initial check-in, then update ``ACKS``
-and ``NEWS.d`` in a subsequent check-in (don't worry about trying to fix the
-original check-in message in that case).
-
-Finally, make sure that the submitter of the
-patch has a CLA in place (indicated by an asterisk following their username
-in the `issue tracker`_ or by the "CLA Signed" label on the pull request).
-If the submitter lacks a signed CLA and the patch is non-trivial, direct them
-to the electronic `Contributor Licensing Agreement`_
-to ensure the PSF has the appropriate authorizations in place to relicense
-and redistribute their code.
-
-
-Contributor Licensing Agreements
---------------------------------
+This page is a step-by-step guide for core developers who need to assess,
+merge, and possibly backport a pull request on the main repository.
+
+Assessing a pull request
+------------------------
+
+Before you can accept a pull request, you need to make sure that it is ready
+to enter the public source tree. Ask yourself the following questions:
+
+* **Was the pull request first made against the** ``master`` **branch?** 
+   The only branch that receives new features is ``master``, the 
+   in-development branch.
+
+* **Are there comments on the pull request?** 
+   Look for explanations about whether the code coverage increased or 
+   stayed the same.
+
+* **Are the changes acceptable?** 
+   If you want to share your work-in-progress code on a feature or bugfix, 
+   then you can open a ``WIP``-prefixed pull request, publish patches on 
+   the `issue tracker <https://bugs.python.org/>`_, or create a public 
+   fork of the repository.
+
+* **Does the check on the pull request show that the test suite passes?** 
+   :ref:`Run the entire test suite <runtests>` to ensure that it passes.
+   Running a single test is not enough since the changes may have unforeseen
+   effects on other tests or library modules.
+
+* **Is the patch in a good state?** 
+   Check :ref:`patch` and :ref:`helptriage` to review what is expected of 
+   a patch.
+
+* **Do the changes meet the requirements of the patch checklist?**
+   :ref:`Run patchcheck <patchcheck>` to perform a quick confidence
+   check on the changes.
```

Comment:
Should core developers run patchcheck? Doesn't CI run patchcheck?

Reply:
@methane Thanks for your feedback. I included this item in the checklist because the current version of this page includes the following section:

```
Patch checklist
'''''''''''''''

You should also :ref:`run patchcheck <patchcheck>` to perform a quick
sanity check on the changes.
```

If core developers don't need to run `patchcheck`, then I have no objection to removing the checklist item.

Reply:
@methane I removed this item from the checklist.

Reply:
I consider this a mistake.  See review.